src/Raven.Server/Documents/DocumentIdWorker.cs

```diff
@@ -404,6 +406,16 @@ public static void ThrowDocumentIdTooBig(string str)
                 nameof(str));
         }
 
+        public static void ThrowRevisionKeyTooBig(string id, string changeVector, bool isTombstone)
+        {
+            var type = isTombstone ? "Revision Tombstone" : "Revision";
+            throw new ArgumentException(
+                $"{type} change vector cannot exceed {RevisionMaxKeySize} bytes, but the change vector was {Encoding.GetByteCount(changeVector)} bytes. " +
+                $"The invalid change vector for {type} '{id}' is '{changeVector}'.{Environment.NewLine}" +
+                $"For more details visit https://ravendb.net/l/ZH9AK4/5.4. {Environment.NewLine}",
```

Comment:
I don't see any actual details related to the exception on `https://ravendb.net/l/ZH9AK4/5.4` docs page. 
Don't we need to write on that page that the unused ids needs to be populated?

Reply:
yep.. the details are not yet documented on the page, but should be :)

Reply:
@reebhub